features/page_objects/sections/evidence_checklist_section.rb

```diff
@@ -1,5 +1,5 @@
 class EvidenceChecklistSection < SitePrism::Section
-  sections :items, ".multiple-choice" do
+  sections :items, ".cc-multiple-choice .govuk-checkboxes__item" do
```

Comment:
This class wraps the group, not the individual check boxes, therefore I'm not sure why this is working, let's have a review together.